src/patternfly/components/Page/page.scss

```diff
@@ -285,6 +295,40 @@
       display: block;
       visibility: visible;
     }
+
+    > .pf-c-button {
+      width: var(--pf-c-page__header-tools-group--m-icons--c-button--Width);
+      height: var(--pf-c-page__header-tools-group--m-icons--c-button--Height);
+      padding: unset;
+      background-color: var(--pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor);
+      border-radius: var(--pf-c-page__header-tools-group--m-icons--c-button--BorderRadius);
+
+      > i {
+        vertical-align: #{pf-size-prem(-.75px)};
+      }
+
+      &:active,
+      &.pf-m-active {
+        --pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor: var(--pf-c-page__header-tools-group--m-icons--c-button--active--BackgroundColor);
+      }
+
+      &:focus,
+      &.pf-m-focus {
+        --pf-c-page__header-tools-group--m-icons--c-button--BackgroundColor: var(--pf-c-page__header-tools-group--m-icons--c-button--focus--BackgroundColor);
+
+        outline-offset: var(--pf-c-page__header-tools-group--m-icons--c-button--focus--OutlineOffset);
```

Comment:
is this necessary? the focus ring seems OK without it.

Reply:
agreed, removed it